setup.py

```diff
@@ -18,8 +18,8 @@
 tests_require = [
     'check-manifest>=0.25',
     'coverage>=4.0',
-    'elasticsearch-dsl<5,>=2.0.0',
-    'elasticsearch<5,>=2.0.0',
+    'elasticsearch-dsl<8,>=2.0.0',
```

Comment:
These two shouldn't be needed at all (i.e. they should be removed from `tests_require`) since the Travis matrix should be installing the correct extras.